### PR TITLE
Do not set CONAN_DOCKER_ENTRY_SCRIPT, but append instead

### PR DESCRIPTION
**Pull request by jmarrec**
This would allow installing system packages for example

Similar to #35

### CODE REVIEW

File: bincrafters/build_autodetect.py [hunk 1]
```diff
@@ -94,9 +94,13 @@ def run_autodetect():
...
     os.system('conan config set general.revisions_enabled=1')
-    os.environ["CONAN_DOCKER_ENTRY_SCRIPT"] =\
-        "conan config set storage.download_cache='{}'; conan config set general.revisions_enabled=1".format(tmpdir)
-    conan_docker_run_options = os.environ.get('CONAN_DOCKER_RUN_OPTIONS','')
+    conan_docker_entry_script = os.environ.get("CONAN_DOCKER_ENTRY_SCRIPT", '')
+    if conan_docker_entry_script:
+        conan_docker_entry_script += "; "
+    conan_docker_entry_script += (
+        "conan config set storage.download_cache='{}'; "
+        "conan config set general.revisions_enabled=1".format(tmpdir))
```
Comment (jmarrec) [inline bincrafters/build_autodetect.py:102]: Simple enough

**Review by Croydon — APPROVED** [review]
LGTM

But out of curiosity, how are you planning to install system requirements?

We are setting `CONAN_SYSREQUIRES_MODE=enabled` so CCI-styled Conan system packages should be able to install system packages

**Comment by jmarrec** [comment]
I'm mainly trying to do a workaround right now. I've solved my problem in the meantime but specifying an older tag via "dockerImage": "conanio/gcc8:1.46.2" so it wouldn't have this commit that removes libxxx-dev: https://github.com/conan-io/conan-docker-tools/pull/446/

Down the line I hope to just fix the issue.

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**